### ISSUE TITLE
archive/mirror of all images

### ISSUE THREAD

**Issue by pokeball99** [ipfspics/server#36]
so we can peer host it like how glop.me does
(sorry if my request is too vague)

**Comment by cloutier** [comment]
Do you mean running a mirror like mentioned in [glop.me FAQ](https://glop.me/faq.html)? That is not supported, we are making an app which will allow you to choose how much space you want to use to help the network, because a backup of everything would be quite large (15 Gb as of today and growing).

**Comment by pokeball99** [comment]
no,what i mean is like the "directory" or hash url for ipfs.pics , or does https://github.com/ipfspics/server/blob/master/db/ipfspics-DB.sql not generate it in that way?

**Comment by pokeball99** [comment]
also,a 15gb is not that much for an archive at all,you could upload the backup to ia(archive.org) if needed,but again,wasn't what i was asking for really

**Comment by cloutier** [comment]
I think it would be a good idea. 

`ipfs name resolve QmdkJZUWnVkEc6yfptVu4LWY8nHkEnGwsxqQ233QSGj8UP | ipfs pin add -r`

Maybe we could publish only the non-controversial content on this hash (no porn, no racist jokes, etc), and have another one with everything. 

@jbenet Could we mirror this on [ipfs community servers](https://github.com/ipfs/infrastructure/blob/master/solarnet/hosts#L20)?

**Comment by jbenet** [comment]
@cloutier maybe we can mirror through pincoop? https://github.com/VictorBjelkholm/pincoop

we can probably pin a lot of the needed stuff in our gateways, but not everything. they dont have a lot of disk space. we'll be adding some storage nodes to the network and have them participate in pincoop-- maybe these could mirror all the images (whether you use pincoop or not). 

> Maybe we could publish only the non-controversial content on this hash (no porn, no racist jokes, etc), and have another one with everything.

Yes, having two lists is helpful for people who want to follow a community code of conduct, etc

**Comment by pokeball99** [comment]
hmm, @jbenet is it just me,or is http://openipfs.xyz down/blank?
(sorry for the kinda off topic question)

**Comment by cloutier** [comment]
@jbenet That looks interesting, I have quite a bit of unused slower hard drives that I could put to some good use. When are you planning to start your coop?

@pokeball99 It has been down for a while it seems, but we can always take the code and roll our own.
